melos.yaml

```diff
@@ -226,12 +217,10 @@ scripts:
 
   # TODO: This is actually stricter than on CI, but we should aim for this
   ci:
-    run: >-
-      melos clean &&
-      melos bootstrap &&
-      melos run build &&
-      melos run test &&
-      melos run coverage &&
-      melos run analyze &&
-      melos run lint
+    steps:
+      - build
+      - test
+      - coverage
+      - analyze
+      - lint
```

Comment:
How come the `clean` and `bootstrap` commands are not explicitly needed in this case anymore?

Reply:
The new steps feature is for chaining _scripts_. Currently it doesn't support _commands_ (made a comment about it [here](https://github.com/invertase/melos/issues/653#issuecomment-2008977433)). Thinking about it a bit, it was perhaps a bit much to require a `clean` anyway, so I removed that, and hence `bootstrap` is not needed either.

Reply:
Wouldn't it be necessary in some cases to clean? Or why do you think it isn't needed?

Reply:
It should be rare, but [`melos clean`](https://melos.invertase.dev/commands/clean) basically brings us closer to a fresh start.

Reply:
We are not currently using the melos _ci_ script on our actual CI. It is intended to go through all (/most) of the motions the github actions does, but locally.. but to be honest it is work in progress.

In some ways it does more, in some ways it does less.

Reply:
Oh yeah, thanks for the reminder